packages/conduit-async/conduit-async.1.2.0/opam

```diff
@@ -0,0 +1,26 @@
+opam-version: "1.2"
+maintainer:   "anil@recoil.org"
+authors:      ["Anil Madhavapeddy" "Thomas Leonard" "Thomas Gazagnaire" "Rudi Grinberg"]
+homepage:     "https://github.com/mirage/ocaml-conduit"
+dev-repo:     "https://github.com/mirage/ocaml-conduit.git"
+bug-reports:  "https://github.com/mirage/ocaml-conduit/issues"
+tags:         "org:mirage"
+license:      "ISC"
+
+build: [
+  ["jbuilder" "subst" "-n" name] {pinned}
+  ["jbuilder" "build" "-p" name "-j" jobs]
+]
+depends: [
+  "jbuilder" {build & >="1.0+beta9"}
+  "core"
+  "ppx_sexp_conv"
```

Comment:
This should have the `{build}` tag

Reply:
@mseri no. see https://github.com/ocaml/opam-repository/issues/11852 for more details -- ppx_sexp_conv actually ships with a runtime dependency

Reply:
Oh yes, you are right. I got confused because it is a build dependency below.